github/Organization.py

```diff
@@ -1326,6 +1333,117 @@ def get_dependabot_alerts(
             url_parameters,
         )
 
+    def get_custom_properties(self) -> PaginatedList[OrganizationCustomProperty]:
+        """
+        :calls: `GET /orgs/{org}/properties/schema <https://docs.github.com/en/rest/orgs/custom-properties#get-all-custom-properties-for-an-organization>`_
+        :rtype: :class:`PaginatedList` of :class:`github.OrganizationCustomProperty.OrganizationCustomProperty`
+        """
+        return PaginatedList(
+            contentClass=github.OrganizationCustomProperty.OrganizationCustomProperty,
+            requester=self._requester,
+            firstUrl=f"{self.url}/properties/schema",
+            firstParams=None,
+        )
+
+    def get_custom_property(self, property_name: str) -> OrganizationCustomProperty:
+        """
+        :calls: `GET /orgs/{org}/properties/schema/{property_name} <https://docs.github.com/en/rest/orgs/custom-properties#get-a-custom-property-for-an-organization>`_
+        :param property_name: string
+        :rtype: :class:`github.OrganizationCustomProperty.OrganizationCustomProperty`
+        """
+        assert isinstance(property_name, str), property_name
+        headers, data = self._requester.requestJsonAndCheck(
+            "GET", f"{self.url}/properties/schema/{urllib.parse.quote(property_name)}"
+        )
+        return github.OrganizationCustomProperty.OrganizationCustomProperty(
+            requester=self._requester,
+            headers=headers,
+            attributes=data,
+            completed=False,
```

Comment:
@jackylamhk any particular reason you decided for `completed=False` here? Looks like `GET {self.url}/properties/schema/{urllib.parse.quote(property_name)}` should return the complete object, right?

Note that `OrganizationCustomProperty` is a `NonCompletableGithubObject`, which cannot be completed anyway.